rules/falco_rules.yaml

```diff
@@ -2948,7 +2958,7 @@
 
 - rule: Redirect STDOUT/STDIN to Network Connection in Container
   desc: Detect redirecting stdout/stdin to network connection in container (potential reverse shell).
-  condition: evt.type=dup and evt.dir=> and container and fd.num in (0, 1, 2) and fd.type in ("ipv4", "ipv6") and not user_known_stand_streams_redirect_activities
+  condition: evt.type in ("dup", "dup2", "dup3") and evt.dir=> and container and fd.num in (0, 1, 2) and fd.type in ("ipv4", "ipv6") and not user_known_stand_streams_redirect_activities
```

Comment:
`dup2` and `dup3` are not supported yet, please refer to [here](https://github.com/falcosecurity/libs/blob/3cc178e06086c126fa539a9d82b3c616796b56a9/driver/event_table.c#L139)

Reply:
Actually, we support `dup2` and `dup3` syscalls although I admit that the question is quite confusing. In a nutshell, these two syscalls are mapped to the simple `dup` event, so using a filter like `evt.type=dup` will also capture dup2 and dup3 calls. `dup2` and `dup3` are just a wrapper for the `dup` syscall, so when you call a `dup2` indeed a `dup` syscall is launched. This is the reason why we could catch all these 3 syscalls with just one event. This does not rule out that in the future we may also add events for `dup2` and `dup3` to make things clearer. To get a pointer to the code you can take a look at this [issue](https://github.com/falcosecurity/libs/issues/150) which dealt with a similar question.